src/components/CustomizedPipeline/CustomizePipelines.tsx

```diff
@@ -26,18 +28,51 @@ import { ComponentKind } from '../../types';
 import { useURLForComponentPRs, enablePAC, disablePAC } from '../../utils/component-utils';
 import { useAccessReviewForModel } from '../../utils/rbac';
 import { ButtonWithAccessTooltip } from '../ButtonWithAccessTooltip';
+import ComponentPACStateLabel from '../Components/ComponentPACStateLabel';
 import { ComponentProps } from '../modal/createModalLauncher';
 
 type Props = ComponentProps & {
   components: ComponentKind[];
 };
 
+const ComponentKebab: React.FC<{ state: PACState; component: ComponentKind }> = ({
+  component,
+  state,
+}) => {
+  const [isOpen, setOpen] = React.useState(false);
+  return (
+    <Dropdown
+      onSelect={() => setOpen(false)}
+      toggle={<KebabToggle onToggle={() => setOpen((v) => !v)} id="toggle-id-6" />}
+      isOpen={isOpen}
+      isPlain
+      position={DropdownPosition.right}
+      dropdownItems={[
+        <DropdownItem
+          key="roll-back"
+          isDisabled={![PACState.error, PACState.pending, PACState.ready].includes(state)}
+          onClick={() => disablePAC(component)}
```

Comment:
Do we need access check for this?

Reply:
Ah yes. Your changes went in. I'll update this.